api/tools/src/main/java/org/pmiops/workbench/tools/LoadDataDictionary.java

```diff
@@ -41,41 +43,48 @@ public CommandLineRunner run(
 
       for (Resource resource : getDataDictionaryExportFiles()) {
         InputStream is = resource.getInputStream();
+        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
Why are we declaring a new mapper for every resource? It looks like you could reuse one (and its format, module, etc) and just pass in the resource's `InputStream` at line 51.